### PR TITLE
allow existing user record during create_user

### PR DESCRIPTION
**Pull request by bnchdrff**
refs https://steemit.slack.com/archives/C5ADW8RQQ/p1521030862000166

### CODE REVIEW

File: src/server/api/general.js [hunk 1]
```diff
@@ -290,41 +290,35 @@ export default function useGeneralApi(app) {
         try {
             if (!emailRegex.test(email.toLowerCase()))
                 throw new Error('not valid email: ' + email);
-            const existingUser = yield findUser({
+            let user = yield findUser({
```
Comment (jnordberg) [inline src/server/api/general.js]: I think we should skip the lookup entirely, unless we are sure that the existing user instance won’t lead to an unrecoverable account

Comment (bnchdrff) [reply src/server/api/general.js]: i'll change it to an upsert

Comment (bnchdrff) [reply src/server/api/general.js]: actually should we remove any rows with matching usernames, and then create a new row for this approved user?

Comment (bnchdrff) [reply src/server/api/general.js]: should we remove rows w/ matching usernames or emails, or just usernames?

Comment (jnordberg) [reply src/server/api/general.js]: I was thinking we just add more users, or is there an unique index?

Comment (bnchdrff) [reply src/server/api/general.js]: no unique index; i changed it to just make a new user record (see commit "just make a new user record" below;))

**Pull request merged**